### DISCUSSION TITLE
Sonoff Dual R3 v2 - Only one channel displayed

### DISCUSSION THREAD

**Discussion by MaartenTorfs** [arendst/Tasmota#18376] [Q&A] [answered]
I recently bought an SONOFF DUAL R3 v2 and flashed tasmota 12.4.0 on it. Used the correct template {"NAME":"Sonoff Dual R3 v2","GPIO":[32,0,0,0,0,0,0,0,0,576,225,0,0,0,0,0,0,0,0,0,0,3200,8128,224,0,0,0,0,160,161,0,0,0,0,0,0],"FLAG":0,"BASE":1} and everything works fine except for showing the values of energytotal, energytoday and energyyesterday for the two channels.  

Also in the telemetry MQTT data only one value of one channel appears
00:16:43.771 MQT: SONOFF/tele/DUALR3_PM10/SENSOR = {"Time":"1970-01-02T00:16:43","Switch1":"ON","Switch2":"ON","ENERGY":{"TotalStartTime":"1970-04-01T13:22:01","Total":0.233,"Yesterday":0.000,"Today":0.233

Only when it reports back its status the two channels appear
00:22:10.855 CMD: EnergyTotal
00:22:10.863 MQT: SONOFF/stat/DUALR3_PM10/RESULT = {"EnergyTotal":{"Total":[0.119,0.117],"Yesterday":[0.000,0.000],"Today":[0.119,0.117]}}

Apparently that was not the case in previous versions (see below). Is there a specific reason why this was changed? Is there somewhere the possibility to adjust this so that still the two channels are shown and become available in the telemetry MQTT data?

thx a lot 

![image](https://user-images.githubusercontent.com/97160146/230863309-04fd13ee-e755-4671-af99-4c4fd18e8b4f.png)

![image](https://user-images.githubusercontent.com/97160146/230862509-7807e1ba-efc3-4586-9f1a-69866f650133.png)

**Comment by arendst** [comment, marked as answer]
For legacy reasons the default is combined totals. Command ``SO129 1`` eneables your request.

Pls read releasenotes, search docs and/or issues for more information.

**Reply by MaartenTorfs** [reply]
thx a lot. I will do

**Comment by HofeBY** [comment]
@arendst
Do you mean Console Command "SetOption129 1" read in [(https://tasmota.github.io/docs/Commands/#setoptions)]

**Reply by arendst** [reply]
Yep. ``SetOption`` may be abbreviated to ``SO``
